pages/price-feeds/create-your-first-pyth-app/evm/_meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "part-1": "Part 1: Create Your first Pyth App on EVM",
```

Comment:
I think this title could be shorter in the menu. It's already nested under "create your first Pyth app" and "EVM", so maybe this should be like "Create a Contract"?

minor thing: we also should decide whether "Your" is captialized or not in titles. I think we're inconsistent about it right now.